furms-web-ui/src/main/java/io/imunity/furms/ui/project/ProjectModelLeaderResolver.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2020 Bixbit s.c. All rights reserved.
+ * See LICENSE file for licensing information.
+ */
+
+package io.imunity.furms.ui.project;
+
+import io.imunity.furms.domain.projects.Project;
+import io.imunity.furms.domain.users.User;
+
+import java.util.List;
+
+public class ProjectModelLeaderResolver {
```

Comment:
I think we don't need this class, lets sync up on Monday, I feel like there was a disconnect.